doc/requirements.txt

```diff
@@ -2,7 +2,7 @@ sphinx-gallery
 sphinx-bootstrap_theme
 numpydoc
 cython
-mne[data]>=0.24
+mne==0.20.1
```

Comment:
It is not a good idea to freeze the MNE version, it should really be avoided. Is this because of bi2012 dependency?

Reply:
Unfortunately yes. It is using `_fetch_file` which doesn t seem available in recent versions of mne:

![image](https://user-images.githubusercontent.com/6229031/159894243-bc86f4d8-ce80-408c-a78b-810919e7b76e.png)

I have just opened an [issue ](https://github.com/plcrodrigues/py.BI.EEG.2012-GIPSA/issues/4) in bi2012, so we can keep trace of it.